post/2023/beyond-sf/index.qmd

```diff
@@ -0,0 +1,301 @@
+---
+title: "R-spatial beyond `{sf}`"
+author: 
+  - name: Josiah Parry
+    affiliation: Esri
+    orcid: 0000-0001-9910-865X
+categories: 
+  - posts
+  - rstats
+tags: [rspatial]
+date: "2023-09-06"
+slug: r-spatial-beyond-sf
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+`{sf}` has transformed the R ecosystem and helped pave the way for wider adoption of geospatial workflows by R users. The R-spatial ecosystem, however, is much bigger than just sf. There are lower level packages such as `{geos}`, `{wk}`, and `{s2}` that provide access to geometries and algorithms outside of the context of sf. These packages are often more performant than sf but require a different frame of mind when using them. 
```

Comment:
In Geocomputation with R we have a convention, following that of the R Journal, of denoting R packages in bold, so this would be 

```suggestion
**sf** has transformed the R ecosystem and helped pave the way for wider adoption of geospatial workflows by R users. The R-spatial ecosystem, however, is much bigger than just sf. There are lower level packages such as **geos**, **wk**, and **s2** that provide access to geometries and algorithms outside of the context of sf. These packages are often more performant than sf but require a different frame of mind when using them.
```

Personally I like the `{pkgname}` way and that is consistent with R itself in its help pages. But we're probably unlikely to change, not least as we've started using the same bold convention in `geocompy`.

Minor issue that we can fix up later..